app/src/main/java/org/mozilla/fenix/Config.kt

```diff
@@ -35,6 +33,12 @@ enum class ReleaseChannel {
             else -> false
         }
 
+    val isRelease: Boolean
+        get() = when (this) {
+            FennecProduction -> true
```

Comment:
`FennecProduction` is org.mozilla.fenix, which is "Nightly" now (all of that will be cleaned up in #12827). So I wonder if that is correct here? It's definitely confusing to have `isRelease` return `true` for a Nightly build? Is that on `master` too? 🤔

Reply:
I thought that was `FenixProduction`? 🤔 I'm confused now. `fennecProduction` looks like it's `org.mozilla.firefox`?? Is the readme outdated again 😭

Reply:
@pocmo could you confirm what I'm seeing? In [the build variants in build.gradle](https://github.com/mozilla-mobile/fenix/blob/master/app/build.gradle#L82) `fennecProduction` is "Firefox Release" (and `fenixProduction` is Nightly).

I'm not sure about where [this nightly/production naming](https://github.com/mozilla-mobile/fenix/blob/master/taskcluster/ci/push-apk/kind.yml#L30) comes from, but it also seems to confirm at least that `fennecProduction` is Firefox Release.

Reply:
@ekager Oh, I'm totally sorry. You are right. I was confused. 🤦